***  

---  

***************

*** 6,14 ****

  Attempting OGRE for universe in build-370614-896976
  RUN: /usr/share/launchpad-buildd/slavebin/update-debian-chroot ['update-debian-chroot', '370614-896976']
  Updating debian chroot for build 370614-896976
! Get:1 http://buildd:secret@ftpmaster.internal gutsy Release.gpg [191B]
  Get:2 http://ftpmaster.internal gutsy Release [65.9kB]
! Get:3 http://user:blah@ftpmaster.internal gutsy/main Packages [1085kB]
  Get:4 http://ftpmaster.internal gutsy/universe Packages [3991kB]
  Fetched 5142kB in 5s (1012kB/s)
  Reading package lists...
--- 6,14 ----

  Attempting OGRE for universe in build-370614-896976
  RUN: /usr/share/launchpad-buildd/slavebin/update-debian-chroot ['update-debian-chroot', '370614-896976']
  Updating debian chroot for build 370614-896976
! Get:1 http://ftpmaster.internal gutsy Release.gpg [191B]
  Get:2 http://ftpmaster.internal gutsy Release [65.9kB]
! Get:3 http://ftpmaster.internal gutsy/main Packages [1085kB]
  Get:4 http://ftpmaster.internal gutsy/universe Packages [3991kB]
  Fetched 5142kB in 5s (1012kB/s)
  Reading package lists...